public/components/main/report_details/report_details.tsx

```diff
@@ -0,0 +1,267 @@
+      description: string,
+      created: string,
+      last_updated: string,
+      source_type: string,
+      source: string,
+      default_file_format: string,
+      report_header: string,
+      report_footer: string,
+      report_type: string,
+      schedule_type: string,
+      schedule_details: string,
+      alert_details: string,
+      channel: string,
+      kibana_recipients: string,
+      email_recipients: string,
+      email_subject: string,
+      email_body: string,
+      report_as_attachment: boolean
+    }
+  }
+
+  const ReportDetailsComponent = (props) => {
+    const {
+      reportDetailsComponentTitle,
+      reportDetailsComponentContent,
+    } = props;
+
+    return (
+      <EuiFlexItem>
+        <EuiDescriptionList>
+          <EuiDescriptionListTitle>
+            {reportDetailsComponentTitle}
+          </EuiDescriptionListTitle>
+          <EuiDescriptionListDescription>
+            {reportDetailsComponentContent}
+          </EuiDescriptionListDescription>
+        </EuiDescriptionList>
+      </EuiFlexItem>
+    );
+  }
+  const created_date = new Date("April 20, 2020 20:32:12");
+
+  const reportDetailsMockMetadata = {
+    report_name: "Daily Sales Report-232o2jsf28492h3rjskfbwjk23",
+    description: "Report Description Here",
+    created: created_date.toString(),
+    last_updated: created_date.toString(),
+    source_type: "Download",
+    source: "dashboard/daily_sales",
+    default_file_format: "PDF",
+    report_header: "--",
+    report_footer: "--", 
+    report_type: "Schedule",
+    schedule_type: "Now",
+    schedule_details: "--",
+    alert_details: "--",
+    channel: "Kibana Reports", 
+    kibana_recipients: "admin",
+    email_recipients: "--",
+    email_subject: "--",
+    email_body: "--",
+    report_as_attachment: false
+  }
+
+
+  export function ReportDetails(props: ReportDetailsRouteProps) {
+    const reportId = props.reportId;
+    // todo: replace values with values from props.reportDetailsMetadata
+    const reportDetailsMetadata = reportDetailsMockMetadata;
+
+    const includeReportAsAttachmentString = reportDetailsMetadata.report_as_attachment ? "True" : "False"
+    return (
+      <EuiPage>
+        <EuiPageBody>
+          <EuiTitle size='l'>
+            <h1>Report Details</h1>
+          </EuiTitle>
+          <EuiPageContent panelPaddingSize={"l"}>
+            <EuiPageHeader>
+              <EuiFlexItem>
+                  <EuiPageHeaderSection>
+                    <EuiTitle>
+                      <h1>{reportDetailsMetadata.report_name}</h1>
+                    </EuiTitle>
+                  </EuiPageHeaderSection>
+                </EuiFlexItem>
+              <EuiFlexGroup justifyContent="flexEnd" alignItems="flexEnd" gutterSize="xs">
+                <EuiFlexItem></EuiFlexItem>
+                <EuiFlexItem></EuiFlexItem>
+                <EuiFlexItem></EuiFlexItem>
+                <EuiFlexItem></EuiFlexItem>
+                <EuiFlexItem></EuiFlexItem>
```

Comment:
why so many empty EuiFlexItem, would you try https://elastic.github.io/eui/#/layout/flex. would you try ```Flex item can specify a proportional width```

Reply:
Empty EuiFlexItems were to make sure the `Archive` `Share` and `Download` texts were spaced properly, had trouble with just specifying with the FlexItem